package.json

```diff
@@ -43,7 +52,21 @@
     "scripts": {
         "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
         "build": "cross-env NODE_ENV=production webpack --mode production --progress --hide-modules --display-error-details",
-        "test": "testcafe chrome,firefox ./tests/*/*.test.js --app 'cross-env NODE_ENV=development webpack-dev-server' --app-init-delay 10000"
+        "test": "testcafe chrome,firefox ./tests/*/*.test.js --app 'cross-env NODE_ENV=development webpack-dev-server' --app-init-delay 10000",
+        "lint-all": "eslint --ext .js,.vue src tests",
+        "lint-all-autofix": "eslint --ext .js,.vue src tests --fix",
+        "lint-check": "eslint-config-prettier .eslintrc.js",
+        "postcommit": "git update-index --again"
     },
-    "version": "1.4.0-dev"
+    "version": "1.4.0-dev",
+    "husky": {
+        "hooks": {
+            "pre-commit": "lint-staged"
+        }
+    },
+    "lint-staged": {
+        "*.{js,json,yaml,json,vue}": [
+            "eslint --ext .js,.vue src tests --fix --quiet --cache"
```

Comment:
I still reckon this could be `yarn lint-all-autofix --quiet --cache` but not a big deal

and `lint-all-fix` could be `yarn lint-all --fix`

It's only annoying if they get out of sync.